app/helpers/foreman_virt_who_configure/configs_helper.rb

```diff
@@ -18,7 +19,8 @@ def hypervisor_username_help_data
         # 'vdsm' => '',
         'hyperv' => _('Account name by which virt-who is to connect to the hypervisor. By default this is <code>Administrator</code>. To use an alternate account, create a user account and assign that account to the following groups (Windows 2012 Server): Hyper-V Administrators and Remote Management Users.'),
         'xen' => _('Account name by which virt-who is to connect to the hypervisor.'),
-        'libvirt' => _('Account name by which virt-who is to connect to the hypervisor. Virt-who does not support password based authentication, you must manually setup SSH key, see <a href="https://access.redhat.com/solutions/1515983">Red Hat Knowledgebase solution How to configure virt-who for a KVM host</a> for more information.')
+        'libvirt' => _('Account name by which virt-who is to connect to the hypervisor. Virt-who does not support password based authentication, you must manually setup SSH key, see <a href="https://access.redhat.com/solutions/1515983">Red Hat Knowledgebase solution How to configure virt-who for a KVM host</a> for more information.'),
+        # 'cnv' => _('') 
```

Comment:
We do not use an account to get this information. We use kubeconfig where we have a certificate.

Reply:
Correct, We load kubeconfig file using environment property. Will take a look at the files you mentioned.